<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialFrame</class>
 <widget class="QWidget" name="SerialFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxCurve24bit">
     <property name="title">
      <string>curve_24bit</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelHead">
        <property name="text">
         <string>head</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditHead"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFoot">
        <property name="text">
         <string>foot</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditFoot"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelLength">
        <property name="text">
         <string>length</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditLength"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnModify">
     <property name="text">
      <string>Modify</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
